sechub-scan/src/main/resources/templates/report/html/scanresult.css

```diff
@@ -167,8 +166,8 @@
 }
 
 .findingsTable td:nth-child(1) {
-	width: 200px;
-	padding-left: 20px;
+	width: 100px;
```

Comment:
Have you tried working with relative values such as [`percentage`](https://developer.mozilla.org/en-US/docs/Web/CSS/percentage) [`em`, `rem`](https://developer.mozilla.org/en-US/docs/Web/CSS/length) etc. The advantage of relative values instead of absolute pixels is they do not depend on the screen resolution and allow to increase or decrease the font size etc.

Reply:
`percentage` wouldn't be a good idea in this case, because the edges of <td> in vertical alignment to another would break more often. Also it would introduce more unnecessary whitespaces in some situations.

A fixed width of e.g. 100px however creates a more stable alignment of vertically aligned <td> elements.

I tried `em` and `rem` but did not see any advantages to `px` in this case. Because in this CSS there is a lot of usage of `px` but none of `em` yet, I would prefer `px` for consistency.